.github/labels.toml

```diff
@@ -1,84 +1,107 @@
-[android]
-color = "25d4dd"
-name = "android"
-description = "The issue relates to Android mobile support."
-
+# These labels are derived from the default Github names and colors,
+# augmented with:
+#   * More details for triage states
+#   * Tags for platform specific issues
+#
+# To sync these labels with a repository
+# 1. Generate a Github Access Token
+# 2. Export a LABELS_USERNAME and LABELS_TOKEN environment variable
+# 3. Install `labels` from PyPI:
+#
+#      pip install labels
+#
+# 4. cd into the .github folder where the labels.toml file is stored
+# 5. Perform a dry run for a specific repo (say beeware/toga):
+#
+#      labels sync -n -f -o beeware -r toga
```

Comment:
```suggestion
#      labels sync -n -o beeware -r toga
```
If I understand the `labels` documentation correctly, `-f` requires an argument, and is unnecessary if labels.toml is in the working directory.